api/tests/acceptance/database/custom-rules/identifier-naming.js

```diff
@@ -0,0 +1,27 @@
+const identifierNaming = {
+  name: 'identifier-naming',
+  docs: {
+    description: 'Identifier columns should follow "id" convention',
+    url: '...',
+  },
+  process({ schemaObject, report }) {
+    const validator = ({ columns, name: tableName }) => {
+      const idColumns = columns.filter((c) => c.isPrimary);
+      if (idColumns.length > 0) {
+        const [idColumn] = idColumns;
```

Comment:
**suggestion :** utiliser la méthode find ?

Reply:
effectivement ca serait plus optimisé 
```suggestion
const idColumns = columns.find((c) => c.isPrimary);
      if (idColumn) {
        const {name} = idColumn;

```